modules.d/95udev-rules/module-setup.sh

```diff
@@ -47,8 +47,21 @@ install() {
         "$moddir/61-persistent-storage.rules"
 
     prepare_udev_rules 59-persistent-storage.rules 61-persistent-storage.rules
-    # legacy persistent network device name rules
-    [[ $hostonly ]] && inst_rules 70-persistent-net.rules
+
+    # debian udev rules
+    inst_rules 91-permissions.rules
+    # eudev rules
+    inst_rules 80-drivers-modprobe.rules
```

Comment:
Do we need to install `61-persistent-storage-edd.rules`, `91-permissions.rules` and `80-drivers-modprobe.rules`? AFAIK we don't ship that.
Other than that, the patch looks ok. Thanks!

Reply:
Oops, thats and cherry-pick artifact.
I will remove it.